examples/scanner_byname.py

```diff
@@ -0,0 +1,30 @@
+"""
+Bleak Scanner
+-------------
+
+
+
+Updated on 2020-08-12 by hbldh <henrik.blidh@nedomkull.com>
+
+"""
+
+import asyncio
+import platform
+import sys
+
+from bleak import BleakScanner
+
+
+if len(sys.argv) != 2:
+    print(f'Usage: {sys.argv[0]} name')
+    sys.exit(1)
+wanted_name = sys.argv[1].lower()
+
+
+async def run():
+    device = await BleakScanner.find_device_by_filter(lambda d, ad : d.name and d.name.lower() == wanted_name)
+    print(device)
+
+
+loop = asyncio.get_event_loop()
+loop.run_until_complete(run())
```

Comment:
I've been using `asyncio.run()` in examples since `loop.run_until_complete()` doesn't always clean up properly if the program is interrupted (even though we technically still support Python 3.6 and `asyncio.run()` requires Python 3.7).

Reply:
Hmm, just copied this from scanner.py. I'm not sure about the exact implications of run_until_complete verses run (I'm still pretty much an async virgin), so I would prefer if you pick this up for all examples at once....

Reply:
Sure, no problem.